src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -15,6 +16,7 @@
     </tr>
     <tr><td><a href="${pageContext.request.contextPath}/drivers">Display All Drivers</a></td></tr>
     <tr><td><a href="${pageContext.request.contextPath}/cars">Display All Cars</a></td></tr>
+    <tr><td><a href="${pageContext.request.contextPath}/cars/all">Display Only My Cars</a></td></tr>
```

Comment:
some missunderstandings between "cars" and "all cars". cars/all could be replaced with owners/cars or cars/current
```suggestion
    <tr><td><a href="${pageContext.request.contextPath}/cars">Display All Cars</a></td></tr>
    <tr><td><a href="${pageContext.request.contextPath}/drivers/cars">Display Only My Cars</a></td></tr>
```